mitiq/utils.py

```diff
@@ -147,6 +149,69 @@ def _are_close_dict(dict_a: Dict[Any, Any], dict_b: Dict[Any, Any]) -> bool:
     return True
 
 
+class CircuitMismatchException(Exception):
+    pass
+
+
+def _generate_pc_circuit(
+        qubits: Iterable,
+        depth: int,
+        gate: EigenGate) -> Circuit:
+    """
+    Generates a circuit which should be the identity. Given a rotation
+    gate R(param), it applies R(2 * pi / depth) depth times, resulting
```

Comment:
What is `param`? The input gate doesn't have an associated parameter, right?

Reply:
I think every non-pathological gate will have period 2pi, but this function doesn't explicitly require this. I would be explicit about this requirement in the docstring.

Reply:
the input gate does have a parameter whether it is explcitly defined by the user or not